test/google_scraping/dashboard/keyword_scraper_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule GoogleScraping.Dashboard.KeywordScraperTest do
+  use GoogleScraping.DataCase, async: true
+
+  alias GoogleScraping.Dashboard.KeywordScraper
+
+  describe "get_search_page_html_for_keyword/1" do
```

Comment:
Shall we assert the error case for this as well? 👍

Reply:
Thanks! Negative test was added here: https://github.com/Nihisil/nimble-certification-elixir/pull/41/commits/03c2b57cd07d71175fed403065e39d1846131ed4